build.rs

```diff
@@ -18,6 +18,12 @@ fn main() {
     }
 
     if cfg!(target_os = "macos") {
+        // Automatically append Homebrew's unixODBC lib path if the environment variable is set
+        if let Ok(unixodbc_lib) = std::env::var("ODBC_SYS_STATIC_PATH") {
```

Comment:
Hm, homebrew sets this variable now, in the formula for odbc2parquet. I wonder though if, we could detect a homebrew installation, independent of the fact if `odbc2parquet` is installed by homebrew.

Maybe running something like: `$(brew --prefix)/lib ` and using that directory would do the trick more robustlyL